### PR TITLE
Feat/new resources

### PR DESCRIPTION
**Pull request by Robert-W**
This PR makes the following changes:

* Added R4(4.0.0) resources
* Updated DSTU2 resources
* Updated STU3 resources
* Added CHANGELOG.md which will double as release notes
* Updated version to 1.0.0 (FINALLY)
* Added `@asymmetrik/sof-graphql-invariant` for cleaner Smart on FHIR support
* Added environment variables for feature flag setting

BREAKING CHANGES (Listed in CHANGELOG.md):

`@asymmetrik/sof-graphql-invariant` depends on a newer version of GraphQL than what was used. GraphQL has been updated here but when upgrading, users may need to remove `node_modules` and reinstall. They will need to run `yarn install` regardless since we added new dependencies.

### CODE REVIEW

**Comment by sureshHARDIYA** [comment]
Could we get this PR merged :) would be nice to have it working.

**Comment by Robert-W** [comment]
@sureshHARDIYA Unfortunately this is probably not going to happen today. Since this PR is so large it is going to take some time to verify things are working the way we want and the newly generated code does not reintroduce old bugs.  We have unit tests and some basic integration tests to verify the code runs fine, but we need to go through and try to make sure things are conformant.

Once @jonterrylee and myself have had a chance to go over all this in detail and feel this is not a regression in any sense, we will get it merged.

If you really need to use this you can check out this branch, `feat/new-resources`. If you do, please let us know if you encounter any issues as that would help us get this merged faster.

File: src/environment.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * WARNING: DO NOT PUT ANY SECRETS IN HERE, THIS IS ONLY INTENDED TO SETUP
+ * SIMPLE FEATURE FLAGS. SECRETS SHOULD BE PART OF YOUR DEPLOYMENT STRATEGY
+ */
+
+if (process.env.NODE_ENV === 'development') {
```
Comment (jonterrylee) [inline src/environment.js:6]: is there a need for defaults?

**Comment by sureshHARDIYA** [comment]
@Robert-W  thank you for notifying. I will check out `feat/new-resources` and test it. Will of course report any issues.

**Review by jonterrylee — APPROVED** [review]

File: src/environment.js (hunk 1 above)
Comment (Robert-W) [reply src/environment.js:6]: Nope, the defaults are already assumed in [`@asymmetrik/sof-graphql-invariant`](https://github.com/Asymmetrik/phx-tools/tree/master/packages/sof-graphql-invariant). Smart on FHIR auth is enabled by default and GraphiQL will also require auth. These are basically just feature flags to control that behavior.

**Pull request merged**